### ISSUE TITLE
podman fact can fail if root is running a rootless socket

### ISSUE THREAD

**Issue by traylenator** [southalc/podman#73]
While its a slightly strange situation to be in steps to reproduce:

```
systemctl start podman.socket
systemctl start --user podman.socket
```
both as root.

Results in:

```
Error: Facter: Error while resolving custom fact fact='podman', resolution='<anonymous>': can't modify frozen String: "Cannot merge \"/run/podman/podman.sock\":String and \"/run/user/0/podman/podman.sock\":String"
```

**Comment by stdietrich** [comment]
For Podman, this is fixed with #71

**Comment by southalc** [comment]
Thanks for reporting and fixing this bug!
